analysis/metrics.py

```diff
@@ -143,3 +144,157 @@ class AggregateMetrics:
     count_metrics: Optional[AggregateErrorMetrics] = None
     privacy_id_count_metrics: Optional[AggregateErrorMetrics] = None
     partition_selection_metrics: Optional[PartitionSelectionMetrics] = None
+
+
+# New dataclasses. They will be used as the output of Utility analysis in the
+# next PRs.
+
+
+@dataclass
+class MeanVariance:
+    mean: float
+    var: float
+
+
+@dataclass
+class ContributionBoundingErrors:
+    """Contains an error breakdown by types of contribution bounding.
+
+    Attributes:
+        l0: max_partition_contributed (aka l0) bounding error. The output of l0
+          bounding is a random variable for each partition.
+        linf: per partition (aka linf) bounding errors. The output of linf
+          bounding is deterministic for each partition.
+        linf_min & linf_max: represents error due to min & max contribution
+          bounding, respectively (only populated for Sum metrics). It is
+          deterministic for each partition.
+    """
+    l0: MeanVariance
+    linf: float
+    linf_min: float
+    linf_max: float
+
+
+@dataclass
+class ValueErrors:
+    """Errors between true and dp metric.
+
+    This class describes breakdown of errors for (dp_value - actual_value),
```

Comment:
nit: Use only `true` or `actual` for consistency (see the line above). I don't have a preference on one over the other, but I think we should have consistency to avoid confusion and allow easier readability.